### ISSUE TITLE
Add support for code coverage

### ISSUE THREAD

**Issue by kospiotr** [danielkec/NodeJS#5]
I recently started working with NodeJS applications using Netbeans IDE. I have simple project that I'm testing with [Intern](https://theintern.github.io/) testing framework. Basically it allows for writing unit and functional tests but also allows to generate code coverage reports. Those reports can be in the following formats:

* **cobertura** - This reporter generates a Cobertura-compatible XML report from collated coverage data.
* **combined** - This reporter stores coverage data generated by the Node.js client in an intermediate file, and then merges in data generated by the WebDriver runner to generate a combined coverage report.
* **lcov** - This reporter generates an lcov.info from collated coverage data that can be fed to another program that understands the standard lcov data format.
* **lcovinfo** - This reporter generates a set of illustrated HTML reports from collated coverage data.

Basically Netbeans understands Cobertura reports for Maven (out of the box) and Ant (via plugin) projects.

Would be great to add support for displaying code coverage from custom report.
Sample NodeJS project can be found [here](https://github.com/kospiotr/hello-js/tree/master/intern).
